CHANGELOG.md

```diff
@@ -20,8 +20,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 Thanks to the people who made this release happen!
 
+ * Martin von Zweigbergk (@martinvonz)
```

Comment:
Please rebase and squash before merging this. Thanks

Reply:
Done.